local-snap.yaml

```diff
@@ -0,0 +1,17 @@
+applications:
+  slurmd:
+    resources:
+      slurm: ./slurm.resource
+  slurmctld:
+    resources:
+      slurm: ./slurm.resource
+  slurmdbd:
+    resources:
+      slurm: ./slurm.resource
+  slurm-configurator:
+    resources:
+      slurm: ./slurm.resource
+  slurmrestd:
+    resources:
+      slurm: ./slurm.resource
+
```

Comment:
we need this file here?

Reply:
it would be nice to have it by default in slurm-bundles, to facilitate development though